src/Modal/Modal.js

```diff
@@ -23,8 +23,9 @@ function getContainer(container, defaultContainer) {
   return ReactDOM.findDOMNode(container) || defaultContainer;
 }
 
-function getOwnerDocument(element) {
-  return ownerDocument(ReactDOM.findDOMNode(element));
+function getOwnerDocument(container, modalInstance) {
+  const node = container || ReactDOM.findDOMNode(modalInstance);
```

Comment:
Why do we need ` || ReactDOM.findDOMNode(modalInstance)`? I would have expected `this.mountNode` to be enough.

Reply:
I think you are correct. I was originally using the `this.props.container` which ended up being troublesome. I'll update this.

Reply:
I'm on it.

Reply:
Damn you're fast! Was just in the middle of fixing :P